psydac/api/grid.py

```diff
@@ -108,15 +94,15 @@ class BasisValues():
     V : FemSpace
        the space that contains the basis values and the spans.
 
-    grid : QuadratureGrid
-        the quadrature grid of the of the discretized domain.
-
     nderiv : int
         the maximum number of derivatives needed for the basis values.
 
     trial : bool, optional
         the trial parameter indicates if the FemSpace represents the trial space or the test space.
 
+    grid : QuaratureGrid, optional
```

Comment:
Typo: QuaratureGrid -> QuadratureGrid

Note the missing `d`

Reply:
done